src/swapnaWJun20/MeetingRoom.java

```diff
@@ -0,0 +1,130 @@
+				takshashilaBookingstatus = true;
+				takshashilaBookingHours = hours;
+			} else {
+				System.out.println(meetingRoom + " room is already booked for " + hours + " Hour");
+				displayAvailableMeetingRoomForBooking();
+			}
+		} else if (meetingRoom.equals(MEETINGROOMS.AGRAFORT)) {
+			if (!agraFortBookingstatus) {
+				System.out.println(meetingRoom + " room is booked for " + hours + " Hour");
+				agraFortBookingstatus = true;
+				agraFortBookingHours = hours;
+			} else {
+				System.out.println(meetingRoom + " room is already booked for " + hours + " Hour");
+				displayAvailableMeetingRoomForBooking();
+			}
+		} else if (meetingRoom.equals(MEETINGROOMS.PRATAPGADH)) {
+			if (!pratapGadhBookingstatus) {
+				System.out.println(meetingRoom + " room is booked for " + hours + " Hour");
+				pratapGadhBookingstatus = true;
+				pratapGadhBookingHours = hours;
+			} else {
+				System.out.println(meetingRoom + " room is already booked for " + hours + " Hour");
+				displayAvailableMeetingRoomForBooking();
+			}
+		}
+	}
+
+	void displayInfo() {
+	//	System.out.println("--------Rooms Booking status----------");
+		if (nalandaBookingstatus)
+			System.out.println("Nalanda room is booked for " + nalandaBookingHours + " hours");
+		if (agraFortBookingstatus)
+			System.out.println("Agrafort room is booked for " + agraFortBookingHours + " hours");
+		if (takshashilaBookingstatus)
+			System.out.println("Takshashila room is booked for " + takshashilaBookingHours + " hours");
+		if (pratapGadhBookingstatus)
+			System.out.println("PratapGadh room is booked for " + pratapGadhBookingHours + " hours");
+	}
+
+	void displayAvailableMeetingRoomForBooking() {
+	//	System.out.println("---------Available for booking------------");
+		if (nalandaBookingstatus && agraFortBookingstatus && takshashilaBookingstatus && pratapGadhBookingstatus) {
+			System.out.println("All meeting rooms are currently booked .");
+		} else {
+			if (!nalandaBookingstatus)
+				System.out.println("Nalanda meeting room is available.");
+			if (!takshashilaBookingstatus)
+				System.out.println("Takshashila meeting room is available.");
+			if (!agraFortBookingstatus)
+				System.out.println("AgraFort meeting room is available.");
+			if (!pratapGadhBookingstatus)
+				System.out.println("PratapGadh meeting room is available.");
+		}
+	}
+
+	public static void main(String[] args) {
+		MeetingRoom meetingRoom = new MeetingRoom();
+		meetingRoom.bookMeetingRoom();
+		System.out.println();
+		meetingRoom.bookMeetingRoom(MEETINGROOMS.AGRAFORT);
+		System.out.println();
+		meetingRoom.bookMeetingRoom(MEETINGROOMS.TAKSHASHILA, 5);
+		System.out.println();
+		meetingRoom.displayInfo();
+		System.out.println();
+		meetingRoom.displayAvailableMeetingRoomForBooking();
+		System.out.println();
+
+		MeetingRoom meetingRoom2 = new MeetingRoom();
+		meetingRoom2.bookMeetingRoom(MEETINGROOMS.NALANDA);
+		System.out.println();
+		meetingRoom.bookMeetingRoom(MEETINGROOMS.PRATAPGADH, 5);
```

Comment:
One observation here the 2nd reference variable should be used. You have used the 1st ref variable.

Reply:
I have updated this change in the code. Thanks :)